csrc/driver_api.cpp

```diff
@@ -5,22 +5,33 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 // clang-format on
-#include <driver_api.h>
-#include <sys_utils.h>
-#include <utils.h>
+#include <iostream>
 
 #include <cuda.h>
+#include <cuda_runtime.h>
 
-#include <iostream>
-
+#include <cuda_utils.h>
+#include <driver_api.h>
 #include <exceptions.h>
+#include <sys_utils.h>
+#include <utils.h>
+
+namespace nvfuser {
+PFN_cuStreamWriteValue32 cuStreamWriteValue32;
+} // namespace nvfuser
 
 namespace {
 
 class CUDADriverAPIDynamicLoader : public nvfuser::LibraryLoader {
  public:
   CUDADriverAPIDynamicLoader() {
     setFilename("libcuda.so");
+
+    // https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#using-the-runtime-api
+    NVFUSER_CUDA_RT_SAFE_CALL(cudaGetDriverEntryPoint(
+        "cuStreamWriteValue32",
+        reinterpret_cast<void**>(&nvfuser::cuStreamWriteValue32),
+        cudaEnableDefault));
   }
```

Comment:
Does it make sense to move this code below to replace the following?

https://github.com/NVIDIA/Fuser/blob/e4cf98e7d0d4e240eea7ef6f23b66bdecc5775a9/csrc/driver_api.cpp#L87-L88

Reply:
Thanks for the suggestions, @zasdfgbnm. I almost got this working but need more time to clean up and verify. Can you approve #4196, the other PR? It's a quick workaround that's safer to land.